### PR TITLE
MWPW-134706 Reevaluate bacom-blog repo

### PR DESCRIPTION
**Pull request by Brandon32**
* Updating from Milo College and changing mountpoint to reevaluate the bacom-blog repo for blog migration.

Resolves: [MWPW-134706](https://jira.corp.adobe.com/browse/MWPW-134706)

**Test URLs:**
- Before: https://main--bacom-blog--adobecom.hlx.page/?martech=off
- After: https://bmarshal-reeveluate-blog--bacom-blog--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Review by meganthecoder — APPROVED** [review]
Am I correct in assuming that other than changing the mount point, these changes are all to match milo?

File: web-test-runner.config.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            }
+          });
+          observer.observe(document.head, { childList: true });
+        </script>
+      </head>
+      <body>
+        <script type='module' src='${testFramework}'></script>
+      </body>
+    </html>`,
+};
```
Comment (JasonHowellSlavin) [inline web-test-runner.config.js:79]: this file has the mjs extension in milo, why change it here?

**Review by JasonHowellSlavin — APPROVED** [review]

File: web-test-runner.config.js (hunk 1 above)
Comment (Brandon32) [reply web-test-runner.config.js:79]: To match milo-college, it is using `"type": "module"` in package.json so mjs is redundant.

**Comment by Brandon32** [comment]
> Am I correct in assuming that other than changing the mount point, these changes are all to match milo?

@meganthecoder To match milo consumers, I using milo-college to see what changed: https://github.com/adobecom/milo-college/compare/53743b743ab818353d392a69be525884bc408239...main

**Pull request merged**